packages/docusaurus-mdx-loader/src/remark/transformImage/__tests__/index.test.ts

```diff
@@ -5,55 +5,49 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import {join, relative} from 'path';
+import path from 'path';
 import remark from 'remark';
 import mdx from 'remark-mdx';
 import vfile from 'to-vfile';
 import plugin from '../index';
 import headings from '../../headings/index';
 
 const processFixture = async (name, options) => {
-  const path = join(__dirname, 'fixtures', `${name}.md`);
-  const file = await vfile.read(path);
+  const filePath = path.join(__dirname, 'fixtures', `${name}.md`);
+  const file = await vfile.read(filePath);
   const result = await remark()
     .use(headings)
     .use(mdx)
-    .use(plugin, {...options, filePath: path})
+    .use(plugin, {...options, filePath})
     .process(file);
 
   return result.toString();
 };
 
 // avoid hardcoding absolute
-const staticDir = `./${relative(process.cwd(), join(__dirname, 'fixtures'))}`;
+const staticDirs = [
+  `./${path.relative(process.cwd(), path.join(__dirname, 'fixtures'))}`,
```

Comment:
What about having a 2nd static dir here?
Test should cover the fact that we look up inside static dirs in a specific order (ie, using for loop, not using non-deterministic `Promise.race` for example)

Reply:
I don't really know... I think undefined behavior is the correct behavior here, because you don't know how `CopyWebpackPlugin` behaves either (though I strongly suppose it's also order-dependent). Having two colliding paths is bad anyways, just like page URLs

Reply:
Even if webpack plugin does something it's a bit unrelated because those assets are loaded in a different way. At least we should test for the ability to find asset in 2 dirs even if there's no override